### PR TITLE
fix: pgfavorite header changes.

### PR DESCRIPTION
**Pull request by fuatguzel**

### CODE REVIEW

File: .ui/library/flAddToCart.cpx [hunk 1]
```diff
@@ -0,0 +1,40 @@
+{
```
Comment (alimoncul) [inline .ui/library/flAddToCart.cpx]: use flCartCheckout component

File: scripts/components/FlCheck.ts [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    }
+    set onToggleChange(value: (toggle: boolean) => void) {
+        this.onTouchEnded = (isInside) => {
+            if (isInside) {
+                this.toggle = !this.toggle;
+                value(this.toggle);
+            }
+        };
+    }
+    private toggleActive() {
```
Comment (alimoncul) [inline scripts/components/FlCheck.ts]: merge this toggle functions into one

File: scripts/components/FlCheck.ts [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+        return this.active;
+    }
+    set toggle(value: boolean) {
+        value ? this.toggleActive() : this.toggleInactive();
+    }
+    get onToggleChange(): (toggle: boolean) => void {
+        return this.__onToggleChange;
+    }
+    set onToggleChange(value: (toggle: boolean) => void) {
+        this.onTouchEnded = (isInside) => {
```
Comment (alimoncul) [inline scripts/components/FlCheck.ts]: use event emitter

File: scripts/components/FlCheck.ts [hunk 4]
```diff
@@ -0,0 +1,55 @@
+import { pushClassNames } from '@smartface/contx/lib/styling';
+import removeClassName from '@smartface/contx/lib/styling/action/removeClassName';
+import FlexLayout from '@smartface/native/ui/flexlayout';
+import ImageView from '@smartface/native/ui/imageview';
```
Comment (alimoncul) [inline scripts/components/FlCheck.ts:1]: remove unused imports

File: .ui/pgFavorites.pgx [hunk 5]
```diff
@@ -55,7 +59,38 @@
...
+			},
+			"degree": 1
+		},
+		{
+			"className": ".sf-flexLayout .flCartCheckout .paddingHorizontal",
+			"hiddenComponent": false,
+			"id": "750e-1819-c16f-2b3e",
+			"props": {
+				"children": [],
+				"name": "flAddToCart",
```
Comment (alimoncul) [inline .ui/pgFavorites.pgx]: gonna remove this

**Review by alimoncul — CHANGES_REQUESTED** [review]

File: .ui/library/flAddToCart.cpx (hunk 1 above)
Comment (fuatguzel) [reply .ui/library/flAddToCart.cpx]: DONE

File: scripts/components/FlCheck.ts (hunk 4 above)
Comment (fuatguzel) [reply scripts/components/FlCheck.ts:1]: DONE

File: .ui/pgFavorites.pgx (hunk 5 above)
Comment (fuatguzel) [reply .ui/pgFavorites.pgx]: DONE

File: scripts/components/FlCheck.ts (hunk 2 above)
Comment (fuatguzel) [reply scripts/components/FlCheck.ts]: DONE

File: scripts/components/FlCheck.ts (hunk 3 above)
Comment (fuatguzel) [reply scripts/components/FlCheck.ts]: Not sure!

**Comment by fuatguzel** [comment]
Favorite items still not working clearly. I noted the problems. But i think OK for now.

File: scripts/components/FlCheck.ts [hunk 6]
```diff
@@ -12,44 +9,43 @@ export default class FlCheck extends FlCheckDesign {
...
-            type: 'pushClassNames',
-            classNames: '.flCheck.inactive'
-        });
-        this.imgCheck.dispatch({
-            type: 'pushClassNames',
-            classNames: '.flCheck-imgCheck.hidden'
-        });
-        this.active = false;
+    private toggleActiveInActive(value: boolean) {
+        if (value) {
```
Comment (alimoncul) [inline scripts/components/FlCheck.ts:29]: if else not needed 

```
this.dispatch({
                type: value? 'removeClassName' :  'pushClassNames',
                className: '.flCheck.inactive'
            });
```
etc

**Pull request merged**